tools/hive-bigquery/custom_exceptions.py

```diff
@@ -0,0 +1,27 @@
+"""Defines custom exceptions"""
+
+
+class ConnectionError(Exception):
```

Comment:
use error python default exceptions `InterruptedException` or the google cloud `NotFound` Exception or exceptions provided by hive and `mysql.connector.Error`.

Reply:
Should I change the base class of this?